### ISSUE TITLE
No colour control l920

### ISSUE THREAD

**Issue by moritzbindewald** [petretiandrea/home-assistant-tapo-p100#352]
Hey there, 

since the new update i color control for my tapo l920 seems to be dead. I can only switch it on or off. No matter what color is chosen, it doesn’t change. Last workin version is v1.3.7. Any way to fix this?

**Comment by ngalatis** [comment]
Had the same issue. Reverted to v1.3.7 while we're waiting on a fix.

**Comment by larueli** [comment]
Hello, I had kind of the same issue with a L530 and found a fix (well for my use case I only have L530 bulbs) maybe you can try it ? https://github.com/petretiandrea/plugp100/issues/80#issuecomment-1407514348

**Comment by petretiandrea** [comment]
Duplicate of #343
